### DISCUSSION TITLE
Coffee

### DISCUSSION THREAD

**Discussion by TheDcol** [sunflower-land/sunflower-land#1141] [Ideas]
I would like to propose the idea/mechanic to BREW coffee to give our farmers a boost to completing tasks/give stamina in future versions. Everyone knows a farmer with energy can work faster!

It would require Coffee Beans and Milk. 

Coffee beans will be grown/produced as a crop, while milk will be a resource supplied by an already planned NFT, Cows.

The idea is to introduce a 'Cafe' building which houses a coffee machine and grinder. You input coffee beans and milk to craft Coffee Beverage (Or we can call it Cappuccino to be a bit more unique). Cappuccino is an item which could give different types of boosts/uses depending on how the team chooses to go. 
The main idea is to give the farmer a boost in production, whether that be in regards to crops or resources. Some ideas are as followed:

- 1 Cappuccino can be consumed to provide a 4-12-24 hour boost to crops/resources claimed or mined.

- 1 Cappuccino can give x stamina to the farmer, stamina is a discussed mechanic for Land Expansion+

- 1 Cappuccino can be sold for SFL.

- 10/100 Cappuccini can be traded to Goblin Town to give a slight discount for NFTS (5-10%)

You could also add other NFTs to attach to the Cafe' such as a sleek coffee machine, an awesome looking grinder, a barista nft, a roasting machine etc.
It pushes players to focus on crops and animals, combining the two to create one item.

I am no 2D artistist and have therefore NO art to provide yet.

Let me know your thoughts :)

**Comment by Parcajr** [comment]
loving the idea, i look foward to this!

**Comment by Yougottabekiddingme123** [comment]
❤ it. Sound like the stamina mechanic in MOM.
